src/main/webapp/synchronizer.js

```diff
@@ -19,26 +19,28 @@ let lastTime;
 
 const TIME_INTERVAL_MS = 100;
 
-/**
- * Starts timer which broadcasts current video time every
- * `TIME_INTERVAL` milliseconds.
- */
-export function startVideoSyncTimer() {
-  window.setInterval(() => {
-    sync(window.videoPlayer.getCurrentTime());
-  }, /* ms= */ TIME_INTERVAL_MS);
-}
+export default class Synchronizer {
+  /**
+   * Starts timer which broadcasts current video time every
+   * `TIME_INTERVAL_MS` milliseconds.
+   */
+  startVideoSyncTimer() {
+    window.setInterval(() => {
+      this.sync(window.videoPlayer.getCurrentTime());
+    }, /* ms= */ TIME_INTERVAL_MS);
+  }
 
-/**
- * Calls functions that seek transcript, and discussion to `currentTime`
- * (number of seconds since start of video), when the `currentTime`
- * changes from the last time this was called.
- */
-function sync(currentTime) {
-  if (currentTime == lastTime) {
-    return;
+  /**
+   * Calls functions that seek transcript, and discussion to `currentTime`
+   * (number of seconds since start of video), when the `currentTime`
```

Comment:
current time is in seconds? Why is that?

Reply:
The video player returns the current time in seconds from the start of the video, the transcript and the discussion areas take the seconds and convert to ms. We will be converting this to ms in the future.

Added a TODO and #267

Reply:
@n-wach is working on this right now

Reply:
Todo sounds great!